### ISSUE TITLE
Emerald Wallet automatically uninstalled in Windows 10 and stop working after being reinstalled

### ISSUE THREAD

**Issue by pcdinh** [ETCDEVTeam/emerald-wallet#541]
Hi,

I installed Emerald Wallet 0.9.0 RC3 2 days ago. I already opened it 2-3 times. Today I attempted to open it again but it was uninstalled.

This happened again and again with Emerald 0.8.0 RC2 dozen times. No application in my PC got this issue. Emerald Wallet is the only. I think that Emerald Wallet violates Microsoft policy or something.

I do search about this issue: https://www.google.com/search?q=Windows+10+application+uninstalled+mysteriously&ie=utf-8&oe=utf-8&client=firefox-b&gfe_rd=cr&dcr=0&ei=48m0Wv6PLKyl8wfupa6wCQ

It looks like a real issue to Emerald Wallet if ETC users don't know why their wallet is not compatible with Windows 10.

However, there is a bigger issue when I install it again. Emerald installer can be found at C:\Users\<user_name>\AppData\Roaming\EmeraldWallet This installer works fine. However, when the installation completes, Emerald Wallet stop working at "Open Wallet"

``
[2018-03-23 16:31:11.494] [info] Starting Emerald Connector...
[2018-03-23 16:31:11.494] [debug] Checking if emerald exists: C:\Users\<user_name>\AppData\Local\Programs\EmeraldWallet\resources\emerald.exe
[2018-03-23 16:31:11.496] [debug] launcherActions - launcher listener:  type SERVICE_STATUS message { mode: { status: 'wrong settings', url: 'http://127.0.0.1:1920' },
  service: 'connector' }
[2018-03-23 16:31:11.497] [debug] launcherActions - launcher listener:  type SERVICE_STATUS message { mode: { status: 'wrong settings', type: 'none', url: '' },
  service: 'geth' }
[2018-03-23 16:31:11.497] [debug] launcherActions - launcher listener:  type SERVICE_STATUS message { mode: 
   { status: 'not ready',
     type: 'remote',
     url: 'https://web3.gastracker.io' },
  service: 'geth' }
[2018-03-23 16:31:11.497] [debug] launcherActions - launcher listener:  type SERVICE_STATUS message { mode: { status: 'not ready', url: 'http://127.0.0.1:1920' },
  service: 'connector' }
[2018-03-23 16:31:11.498] [debug] OK: emerald exists:  C:\Users\<user_name>\AppData\Local\Programs\EmeraldWallet\resources\emerald.exe
[2018-03-23 16:31:11.499] [debug] Emerald bin: C:\Users\<user_name>\AppData\Local\Programs\EmeraldWallet\resources\emerald.exe, args: import,--chain=mainnet,--all,C:\Users\<user_name>\AppData\Roaming\.emerald\mainnet\keystore\
[2018-03-23 16:31:12.264] [debug] Emerald execution status: 1
[2018-03-23 16:31:12.267] [debug] Emerald bin: C:\Users\<user_name>\AppData\Local\Programs\EmeraldWallet\resources\emerald.exe, args: server
[2018-03-23 16:31:12.352] [debug] Emerald log target dir: C:\Users\<user_name>\AppData\Roaming\EmeraldWallet\logs
[2018-03-23 16:31:12.397] [debug] [emerald] [INFO]	Missed `--chain` argument. Use default: `mainnet`

[2018-03-23 16:31:12.404] [debug] [emerald] [ERROR]	Command execution error: KeyFile storage error: Can't create database Keyfile storage

[2018-03-23 16:31:12.411] [error] Emerald Connector process exited with code: 1
[2018-03-23 16:31:13.481] [debug] on get-status
``

**Comment by shanejonas** [comment]
Might be related to this: https://github.com/ETCDEVTeam/emerald-wallet/issues/434, do you have any symbols in your username?

**Comment by pcdinh** [comment]
No, pure ASCII characters (a-z). No space, no dot, no uppercase ...

**Comment by shanejonas** [comment]
you can try the troubleshooting section here to reset some settings: https://github.com/ETCDEVTeam/emerald-wallet#troubleshooting

Also try updating to the latest version of emerald wallet

**Comment by pcdinh** [comment]
> Also try updating to the latest version of emerald wallet

@shanejonas The issue occurs in the latest version of emerald wallet
